### DISCUSSION TITLE
Syntax highlighting does not work for C "goto" statement; statement is not highlighted at all

### DISCUSSION THREAD

**Discussion by aaronmdjones** [github-linguist/linguist#6853] [Classification] [answered]
From a GitHub pull request "Files Changed" tab:

![Screenshot from 2024-05-24 16-01-18](https://github.com/github-linguist/linguist/assets/3105118/839e3561-9c53-45ff-8cbf-be8dcd9a226f)

The goto statement has no highlighting. It should be highlighted in the same manner as other statements like "return" immediately following it, and "if" and "for" immediately preceding it.

**Comment by lildude** [comment, marked as answer]
We have a point for this in our troubleshooting docs 😉 [There's a problem with the syntax highlighting of a file](https://github.com/github-linguist/linguist/blob/master/docs/troubleshooting.md#theres-a-problem-with-the-syntax-highlighting-of-a-file)
